pipelines/build/common/build_base_file.groovy

```diff
@@ -190,6 +227,11 @@ class Builder implements Serializable {
         return overrideDocker
     }
 
+    /*
+    Retrieves the dockerImage attribute from the build configurations.
+    This specifies the docker hub org and image to pull or build in case we don't have one stored in this repository.
```

Comment:
docker hub -> DockerHub

It's styled as `DockerHub` on line 252 so might as well be consistent.